gcp_variant_transforms/vcf_to_bq_common.py

```diff
@@ -55,6 +55,8 @@ def parse_args(argv, command_line_options):
       specifies the options that will be added to parser.
   """
   parser = argparse.ArgumentParser()
+  # TODO(bashir2): The next line is not needed; instead of type='bool' use
```

Comment:
We actually defined this as it's more robust than the default one (i.e. works with both `True` and `true`; common in python as people are used to "True"). Please continue to use it unless there is another issue about it that I'm unaware of.

Reply:
Hmm okay but is it actually common that people provide value for boolean options? I thought the pattern is usually --boolean_flag with no value.

Anyway I removed the TODO per your comment. The reason for not liking it was that I should do the same wherever I want to parse arguments (e.g., in the new AnnotationOptionsTest).